eng/pipelines/partner-release.yml

```diff
@@ -0,0 +1,46 @@
+trigger: none
+pr: none
+
+resources:
+  repositories:
+    - repository: azure-sdk-build-tools
+      type: git
+      name: internal/azure-sdk-build-tools
+      ref: refs/tags/azure-sdk-build-tools_20200618.4
+
+variables:
+  BuildToolScripts: $(Pipeline.Workspace)/azure-sdk-build-tools/scripts
+  Artifacts: $(Pipeline.Workspace)/artifacts
+
+pool:
+  vmImage: 'windows-latest'
```

Comment:
Lets use `windows-2019` here. I'd like us to be in control of when we move the image forward so we can at least test it.